packages/schematics/angular/environments/index.ts

```diff
@@ -0,0 +1,113 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { Rule, SchematicsException, chain } from '@angular-devkit/schematics';
+import { AngularBuilder, TargetDefinition, updateWorkspace } from '@schematics/angular/utility';
+import { posix as path } from 'path';
+import { Schema as EnvironmentOptions } from './schema';
+
+const ENVIRONMENTS_DIRECTORY = 'environments';
+const ENVIRONMENT_FILE_CONTENT = 'export const environment = {};\n';
+
+export default function (options: EnvironmentOptions): Rule {
+  return updateWorkspace((workspace) => {
+    const project = workspace.projects.get(options.project);
+    if (!project) {
+      throw new SchematicsException(`Project name "${options.project}" doesn't not exist.`);
+    }
+
+    const buildTarget = project.targets.get('build');
```

Comment:
Can you also add the same thing for `server` target? It's important that the file replacements are added to both.